src/index.ts

```diff
@@ -6,3 +6,5 @@ export type {Config} from './config.js';
 export {normalize} from './normalize/index.js';
 export type {TopLevelSpec} from './spec/index.js';
 export * from './util.js';
+
+import './types_unstable.js';
```

Comment:
Do we need to have this import if we already have this in the package.json?

```
    "./types_unstable": {
      "types": "./build/types_unstable.d.ts"
    },
```

Reply:
I think I needed this for rollup to produce the file. It shouldn't change the default bundle.